manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LibRetro.RetroArch.Nightly
+PackageVersion: Nightly
+PackageLocale: en-US
+Publisher: LibRetro

```

```suggestion
Publisher: Libretro
```
This must match Control Panel.